metabase/metabase
# Migrate `graph.metrics` sync to use QP metadata

Similar to https://github.com/metabase/metabase/pull/38995, migrate the whole viz settings sync process to rely on the QP metadata and not MLv2.

How to verify:
- Create a question with an aggregation and a breakout, e.g. orders -> Count by Created At (month)
- Visualize as a line chart
- Change some viz settings
- Add a new breakout
- A new series in the chart should appear

We already have a lot of tests for this, not adding a new test here.
